modules/generation_parameters_copypaste.py

```diff
@@ -282,6 +286,15 @@ def parse_generation_parameters(x: str):
 
     restore_old_hires_fix_params(res)
 
+    # If the prompt is being pasted from another tab, respect the "Send
+    # seed/size to other tab" options
+    if source_tabname and source_tabname != "pnginfo":

```

Comment:
Perhaps this is just me, but I don't see the benefit of having send seed/size options being respected everywhere _except_ pnginfo, pnginfo should be included in this with that setting enabled imo.

For example, in my use case I generate at a low resolution in bulk, then I would go through manually and find the generations that stand out and are worth exploring, edit the image in PS to fix hands or oddities, finally I then drag it into pnginfo, click send to img2img and then upscale the image to the correct resolution i need. This mimics the later part of txt2img with upscale fix exactly.

Without pnginfo honouring this setting each time I add a new image and send the gen params it uses the existing low resolution width and height, it also inserts the same seed the low res sample was generated with (using the same seed to upscale doesn't upscale correctly), so in order to avoid these you have to manually click random seed and edit the height and width each time. That doesn't make any sense to me if I have enabled the settings to prevent sending that info